.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,37 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/javascript-node
+{
+  "name": "Node.js",
+  // Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
```

Comment:
A lot of the comments in this file feel boilerplate-ey. Like this comment feels like it is trying to tell me how to write a `devcontainer.json` in the abstract, not describing the `devcontainer.json` you've written here.

Reply:
It is boiler plate (generated from vscode), and so it is talking in the abstract. Happy to prune it away, but figured I'd leave it in until told otherwise